jans-linux-setup/jans_setup/static/scripts/get_agama_lab_projects.py

```
@@ -17,7 +37,12 @@ def get_projects():
                 for asset in result['assets']:
                     if asset['name'].endswith('.gama'):
                         downloads.append({'repository-name': repo_name, 'description': repo['description'], 'download-link': asset['browser_download_url']})
+
+        with open(cache_file, 'w') as w:
+            json.dump(downloads, w, indent=2)
+
         return {'result': True, 'projects': downloads, 'error': None}
```

Ruff (0.14.2)

Consider moving this statement to an `else` block

(TRY300)